include/socket.hpp

```diff
@@ -493,14 +719,38 @@ class Socket final {
 	// recv method calls
 
 
+	/** @method setOpt
+	* @access public
+	* Set a socket option from net::SF::opt for Socket using object of type net::SF::sockOpt.
+	* 
+	* @param {net::SF::opt} _opType Option to set for Socket.
+	* @param {net::SF::sockOpt} _opValue ___________			!!Improve This!!.
+	*/
 	void setOpt(SF::opt, SF::sockOpt) const;
+
+	/** @method getOpt
+	* @access public
+	* Get value of some socket option for Socket.
+	* 
+	* @param {net::SF::opt} _opType Option of Socket whose value to get.
+	*/
 	SF::sockOpt getOpt(SF::opt) const;
 
+	/** @method stop
+	* @access public
+	* Shutdown Socket using net::SF::shut.
+	* 
+	* @param {net::SF::shut} s Option specifying which side of connection to shutdown for Socket.
+	*/
 	void stop(SF::shut s) const noexcept
```

Comment:
Please make it `_s` instead of `s` both in documentation and function definition.